packages/components/src/ui/context/index.js

```diff
@@ -1,2 +1,8 @@
 export { ComponentSystemProvider } from './component-system-provider';
 export { withNext } from './with-next';
+export {
```

Comment:
They aren't exposed outside or `@wordpress/components` for now, right?

Reply:
Nope! None of these are exposed in `components/src/index.js` as that file explicitly only exports `ComponentsSystemProvider` and `withNext`, both as experimental anyway.

https://github.com/WordPress/gutenberg/blob/bebfefb3e5d9ecb19692d0f40bb3fc77baab823c/packages/components/src/index.js#L153-L157